contributors_guide.rst

```diff
@@ -0,0 +1,376 @@
+Contributor's Guide
+===================
+
+Note: This guide is still a work in progress.
```

Comment:
I think this can be removed.  This guide is sufficient and can always be updated in the future.